src/components/hub/elements/MessagesList.js

```diff
@@ -56,74 +56,49 @@ export default function MessagesList({ scrollRef }) {
       return;
     }
 
-    api
-      .messageList({
-        cid: selectedCID,
-        limit: +process.env.REACT_APP_MESSAGES_COUNT_TO_PRELOAD,
-        updated_at: { lt: messages[0].created_at },
-      })
-      .then((arr) => {
-        if (!arr.length) {
-          needToGetMoreMessage.current = false;
-          return;
-        }
-
-        const messagesIds = arr.map((el) => el._id).reverse();
-        needToGetMoreMessage.current = !(
-          messagesIds.length < +process.env.REACT_APP_MESSAGES_COUNT_TO_PRELOAD
-        );
+    const processMessages = (arr) => {
+      indexedDB.insertManyMessages(arr);
```

Comment:
let's make `messagesService ` responsible for communicating with `indexedDB `, 
so `MessagesList` (UI component) should not know which DB is used